src/GameLogic.cpp

```diff
@@ -21,30 +22,43 @@ PlayerActor GameLogic::getPlayer(){
   return player;
 }
 
+void GameLogic::EPressed(){
+	Etracker++;
```

Comment:
why do we need an Etracker and a tracker? Couldn't we just use the tracker for e pressing?